### PR TITLE
Initial version of GitHub Templates

### PR DESCRIPTION
**Pull request by dizzzz**
- Simplify the landing page of the GitHub repo
- The first step to get the PRs and GitHub Issues on a higher level. 

We'll rework the files for sure in the near future.

### CODE REVIEW

**Comment by dizzzz** [comment]
Squash and merge please :-)

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+
+Although the following are taken from our [Developer Manifesto](http://www.exist-db.org/exist/apps/doc/devguide_manifesto.xml "eXist Project Developer Manifesto") and [Code Review Guide](http://www.exist-db.org/exist/apps/doc/devguide_codereview.xml "eXist Project Code Review Guide"), the main things that get a Pull Request accepted are:
+
+- **Only change what you need to.** If you must reformat code, keep it in a separate commit to any syntax or functionality changes.
+- **Test.** If you fix something prove it, write a test that illustrates the issue before you fix the issue and validate the test. If you add a new feature it needs tests, so that we can understand its intent and try to avoid regressions in future as much as possible.
+- **Make sure the appropriate licence header appears at the top of your source code file.** We use [LGPL v2.1](http://opensource.org/licenses/LGPL-2.1 "The GNU Lesser General Public License, version 2.1") for eXist and *strongly* encourage that, but ultimately any compatible [OSI approved license](http://opensource.org/licenses "Open Source Licenses") without further restrictions may be used.
```
Comment (shabanovd) [inline CONTRIBUTING.md]: should we force to use [this header](https://github.com/eXist-db/exist/blob/develop/src/template_java.txt)?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+
+
+Syncing a Fork
+--------------
+Your fork will eventually become out of sync with the upstream repo as others contribute to eXist. To pull upstream changes into your fork, you have two options:
+
+1. [Merging](https://help.github.com/articles/syncing-a-fork).
+2. Rebasing.
+
+Rebasing leads to a cleaner revision history which is much easier to follow and is our preferred approach. However, `git rebase` is a very sharp tool and must be used with care. For those new to rebase, we would suggest having a backup of your local (and possibly remote) git repos before continuing. Read on to learn how to sync using rebase.
```
Comment (shabanovd) [inline CONTRIBUTING.md]: another problem with `git rebase` that it destroy context of discussion (comments) on github

File: CONTRIBUTING.md (hunk 1 above)
Comment (ljo) [inline CONTRIBUTING.md]: Yes, otherwise we have to add it afterwards.

Comment (dizzzz) [inline CONTRIBUTING.md]: this text was copied as-is from the original README.md ; the link from @shabanovd was configured to work with netbeans (the IDE that knows of resources inside the source tree :-) )

File: CONTRIBUTING.md (hunk 2 above)
Comment (dizzzz) [inline CONTRIBUTING.md]: I did not know. But this text was copied as-is from the original README.md ; if it is not the best text/procedure we need to update it, preferable in a new PR.

**Pull request merged**